docs/cloud-tutorials/application-management.md

```diff
@@ -5,59 +5,59 @@ description: Learn how to manage DBOS Cloud applications
 pagination_prev: null
 ---
 
-In this guide, you'll learn how to manage applications in DBOS Cloud.
-
-
 ### Deploying Applications
 
-To deploy your application to DBOS Cloud, run this command in its root directory:
+To deploy your application to DBOS Cloud or update an existing application, run this command in its root directory:
 
 ```shell
 dbos-cloud app deploy
 ```
 
-Your application is deployed using the `name` in its `dbos-config.yaml`.
-Application names should be between 3 and 30 characters and must contain only lowercase letters and numbers, dashes (`-`), and underscores (`_`). Application names are unique within an [organization](account-management#organization-management).
-
-The first time you deploy an application, you are prompted to choose to which [database instance](../cloud-tutorials/database-management.md) to connect your app, or to provision one if you have none.
-Multiple applications can connect to the same database instance (server)&mdash;they are deployed to isolated databases on that server.
-
 Each time you deploy an application, the following steps execute:
 
 1. **Upload**: An archive of your application folder is created and uploaded to DBOS Cloud. This archive can be up to 500 MB in size.
-2. **Configuration**: Your application's dependencies are installed and the application is built.
-    - In Python, dependencies are loaded from `requirements.txt`. In TypeScript, they are loaded from `package-lock.json`, or from `package.json` if the former is not present. The maximum size of your app after all dependencies are installed is 2 GB.
-    - In TypeScript, your application is built using `npm run build`.
-    - After dependencies are installed, all database migrations specified in your `dbos-config.yaml` are run on your cloud database.
-3. **Deployment**: Your application is deployed to a number of [Firecracker microVMs](https://firecracker-microvm.github.io/) with 1vCPU and 512MB of RAM by default.
-    - DBOS Pro subscribers can [configure](../cloud-tutorials/cloud-cli#dbos-cloud-app-update) the amount of memory allocated to each microVM.
-
-MicroVMs expect your application to serve requests from port 8000 (Python&mdash;the default port for FastAPI and Gunicorn) or 3000 (TypeScript&mdash;the default port for DBOS Transact and Koa).
-
-After your application is deployed, the URL of your deployed application is printed.
-This URL is of the form `https://<username>-<app-name>.cloud.dbos.dev/`.
-If your account is part of an [organization](./account-management.md#organization-management), organization name is used instead of username.
```

Comment:
I'd keep this information around. We've seen users having issues finding it in the past. Doesn't hurt to repeat a bit & how it is constructed is useful.